desktop_version/src/titlerender.cpp

```diff
@@ -2933,7 +2933,15 @@ void towerrender(Graphics& dwgfx, Game& game, mapclass& map, entityclass& obj, U
         dwgfx.flashlight();
     }
 
-    dwgfx.render();
+    if (game.screenshake > 0 && !game.noflashingmode)
+    {
+        game.screenshake--;
```

Comment:
Does this need to be decremented even if noflashingmode is on? Wondering if this would cause a screenshake to build up until the mode was disabled, triggering a seemingly random screenshake.

Reply:
This is the exact same code used everywhere else, and is also used for screen flashing (making the screen white).

It might be a good idea to decrement both the shake and flash timers even if screen effects are disabled, but that would be another PR. Same thing with resetting screen shaking and flashing when exiting to the menu.

Reply:
Makes sense, as long as it’s consistent!